src/pages/ImportData.js

```diff
@@ -108,7 +108,7 @@ const ImportData = () => {
         notify('success', 'Data imported successfully.', { timeout: 3000 })
       }]
     )
-    Ajax().Metrics.captureEvent(Events.workspaceDataImport)
+    Ajax().Metrics.captureEvent(Events.workspaceDataImport, { 'Workspace Name': name, 'Workspace Namespace': namespace })
```

Comment:
The strings 'Workspace Name' and 'Workspace Namespace' are repeated a bunch here. Could we either extract them as string constants, or create a helper function that takes a workspace returns that object (which we could destructure into the event metadata?)

I probably prefer the second one somewhat, because it would make it easier to add workspace metadata to all workspace metrics moving forwards?

Reply:
Seconded, I like the helper function idea.

Also, I think we should use camelcase identifiers for properties, e.g. `workspaceName`. I'm pretty sure there's a feature in Mixpanel UI to create friendly name mappings, so it would be better to keep the low level representation more 'data-like'.